cmake/KokkosConfigCommon.cmake.in

```diff
@@ -91,27 +89,16 @@ function(kokkos_check)
   endforeach()
   set(KOKKOS_CHECK_SUCCESS TRUE)
   foreach(arg ${REQUESTED_ARGS})
-    # Define variables named after the required arguments that are provided by
-    # the Kokkos install.
+    set(missing)
     foreach(requested ${KOKKOS_CHECK_${arg}})
-      foreach(provided ${Kokkos_${arg}})
-        string(TOUPPER ${requested} REQUESTED_UC)
-        string(TOUPPER ${provided} PROVIDED_UC)
-        if(PROVIDED_UC STREQUAL REQUESTED_UC)
-          string(REPLACE ";" " " ${requested} "${KOKKOS_CHECK_${arg}}")
-        endif()
-      endforeach()
+      string(TOUPPER "${requested}" REQUESTED_UC)
+      if(NOT "${REQUESTED_UC}" IN_LIST Kokkos_${arg})
```

Comment:
This doesn't convert the values in `Kokkos_${arg}` to uppercase. I would prefer if we keep that conversion (unless enforced elsewhere).

Reply:
Done in 1b973bf7fb1ac503d53919fef9ba2a4256d37274